deprecated_charts/px-license-server/README.md

```diff
@@ -10,7 +10,7 @@ PX-Central is a unified, multi-user, multi-cluster management interface. Using P
 ```console
 $ kubectl label node <NODE_NAME> px/ls=true
 ```
-- For openshift cluster:
```

Comment:
We should also remove deprecated_charts.
Can we test new install with new changes ?

Reply:
Done and tested !



![Screenshot from 2023-11-10 12-47-49](https://github.com/portworx/helm/assets/101168875/e73040cd-4d4d-4d0f-b313-1f2182cdb782)